docs/variables/count.md

```diff
@@ -12,7 +12,7 @@ Returns the current number that is stored in a custom counting variable.
 
 #### Parameters
 
-This variable takes **one** *required* parameter that is a custom counting variable name. Defaults to the behavior of `$(count)` if no parameter is provided, and returns as well as increments the command usage count by `1`.
+This variable takes **one** *required* parameter that is a custom counting **variable name**. Defaults to the behavior of `$(count)` if no parameter is provided, and returns as well as increments the command usage count by `1`.
```

Comment:
This is an optional parameter for keywords and commands.

It also only increments if you don't pass an argument (little confusing I know, but backwards compatibility)

Reply:
Yeah that's a typo and should have been *optional* instead of *required*. I'll change it and update the overall description a bit so it reflects that this behavior occurs in commands and keywords.